src/Generator.Rewrite/Program.cs

```diff
@@ -147,7 +147,7 @@ private void Rewrite(string file, string keyfile, IEnumerable<string> options)
                     assembly.Write(write_params);
                 }
             }
-            catch (InvalidOperationException inex)
+            catch (InvalidOperationException)
```

Comment:
Better to make use of the exception and add the message to the console output. I didn't do it last time around, but I should have.